src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructureWithWorkOrder.java

```diff
@@ -107,7 +108,17 @@ public void initiate()
                 }
 
                 worker.getCitizenChatHandler().sendLocalizedChat(COM_MINECOLONIES_COREMOD_ENTITY_BUILDER_BUILDSTART, job.getStructure().getBluePrint().getName());
-
```

Comment:
Wouldn't it be better to put this at the creation of the workorder and don't even send it to the builder?

Reply:
I suppose it could be. I will move it over there.

Reply:
Perfect! Nice catch

Reply:
Fixed in the latest push.